meteor-app/imports/api/attendances/attendances.js

```diff
@@ -0,0 +1,26 @@
+import { Mongo } from 'meteor/mongo';
+
+const Attendances = new Mongo.Collection('attendances');
+
+Attendances.attachSchema(
+  new SimpleSchema({
+    roleId: {
+      type: String,
+      regEx: SimpleSchema.RegEx.Id,
+      label: "Attending Person ID",
+      optional: false
+    },
+    timeIn: {
+      type: Number,
```

Comment:
ok that cool, i used timestamp as i thought it was easier to determine duration of time spent, upon logging out. this will be an interesting function to write. (Shotgun doing this)

https://trello.com/c/ykoh5Y2w